sorc/vcoord_gen.fd/docs/user_guide.md

```diff
@@ -3,16 +3,15 @@
 
 # Introduction
 
-Generates hybrid coordinate parameters from fields such as surface
-pressure, model top and the number of vertical levels. Outputs the
-'ak' and 'bk' parameters used by the forecast model to define the
-hybrid levels.
+The vcoord_gen program driver.F90 generates hybrid coordinate
```

Comment:
Do you want to add a link to driver.F90 here?

Reply:
I see.  The file extension is lower case - .f90

Reply:
Sorry @edwardhartnett my comment was not clear.  This link to driver.F90 did not work for me.   The vcoord_gen driver is driver.f90 (with lowercase f90).

Reply:
Here is what I see: https://www.emc.ncep.noaa.gov/jcsda/ggayno/doxygen/html/vcoord_gen/index.html

Reply:
OK, good catch, thanks. I fixed...